packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -12,6 +12,8 @@ import { KnownCreateElementDataIndex, ListType } from 'roosterjs-editor-types';
 
 const orderListStyles = [null, 'lower-alpha', 'lower-roman'];
 
+const MARGIN_BASE = '0in 0in 0in 0.5in';
+const NEGATIVE_MARGIN = '-.25in';
```

Comment:
who will generate this value? What if other negative value?

Reply:
These values are set when trying to outdent the first list item and it is the first item in the list, I checked at what Word is doing to perform this action and this is the styles that they add to the List.

Reply:
To give more context, this is what we are trying to achieve, instead of removing the bullet, we outdent using this margin.
![prevent removal](https://user-images.githubusercontent.com/8291124/161145581-e0e9f048-1951-4a50-8b52-523503be2592.gif)